### PR TITLE
feat: add release notes for timeouts

### PR DESCRIPTION
**Pull request by loujaybee**

### CODE REVIEW

File: src/lib/contents/changelog/timeouts.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+alt: Input box for timeouts set in a users preferences page.
+---
+
+<script>
+  import Contributors from "$lib/components/changelog/contributors.svelte";
+</script>
+
+You can now set your workspace timeout up to 24 hours using the command line, direct in your favorite editor or IDE or configuring a global default! Setting a configurable timeout is useful if you're working on a long running task, need to leave your workspace open for a while and then revisit it, or whilst collaborating or sharing previews with others.
+
+To try out this feature, simply type: `gp timeout set 1h` in your running workspace to set the timeout for the currently running workspace. Or, go to your [user preferences](https://gitpod.io/user/preferences) to set the default value for all future timeouts.
```
Comment (filiptronicek) [inline src/lib/contents/changelog/timeouts.md:16]: Maybe we can be explicit about what this command does? I don't really like the wording in my recommendation but just to illustrate what I am looking for.

```suggestion
To try out this feature, simply type: `gp timeout set 1h` in your running workspace to set the timeout for the currently running workspace to 1 hour. Or, go to your [user preferences](https://gitpod.io/user/preferences) to set the default value for all future timeouts.
```

File: src/lib/contents/changelog/timeouts.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+image: timeouts.jpg
+ogImage: timeouts.jpg
+alt: Input box for timeouts set in a users preferences page.
+---
+
+<script>
+  import Contributors from "$lib/components/changelog/contributors.svelte";
+</script>
+
+You can now set your workspace timeout up to 24 hours using the command line, direct in your favorite editor or IDE or configuring a global default! Setting a configurable timeout is useful if you're working on a long running task, need to leave your workspace open for a while and then revisit it, or whilst collaborating or sharing previews with others.
```
Comment (filiptronicek) [inline src/lib/contents/changelog/timeouts.md:14]:
```suggestion
You can now set your workspace timeout up to 24 hours using the command line, direct in your favorite editor or IDE or configure a global default! Setting a configurable timeout is useful if you're working on a long running task, need to leave your workspace open for a while and then revisit it, or whilst collaborating or sharing previews with others.
```

**Review by Siddhant-K-code — APPROVED** [review]

File: src/lib/contents/changelog/timeouts.md (hunk 1 above)
Comment (loujaybee) [reply src/lib/contents/changelog/timeouts.md:16]:
```suggestion
To try out this feature, type: `gp timeout set 1h` to set the timeout for the currently running workspace to 1 hour. Or, go to your [user preferences](https://gitpod.io/user/preferences) to set the default value for all future timeouts.
```

**Review by filiptronicek — APPROVED** [review]
LGTM!

**Comment by loujaybee** [comment]
Blocked depending on: 

- https://github.com/gitpod-io/gitpod/issues/16897

File: src/lib/contents/changelog/timeouts.md [hunk 3]
```diff
@@ -0,0 +1,26 @@
+---
+title: Flexible Timeouts
```
Comment (loujaybee) [inline src/lib/contents/changelog/timeouts.md:2]:
```suggestion
title: Introducing: Flexible Timeouts 🥳  !
```

**Pull request merged**